root/layout/components/sidebar/CollectionLinks.js

```diff
@@ -20,33 +20,37 @@ type Props = {|
 |};
 
 const CollectionLinks = ({$c, entity}: Props) => {
-  const allCollections = $c.stash.all_collections;
-  if (!$c.user_exists || !allCollections) {
+  const numberOfCollections = $c.stash.number_of_collections || 0;
+  if (!$c.user_exists || numberOfCollections < 1) {
```

Comment:
Perhaps we should remove the `numberOfCollections` check? Then at least an "Add to a new collection" link would be shown, which still seems useful.

Reply:
Sure, seems reasonable. Done.